src/nrtk_explorer/widgets/nrtk_explorer.py

```diff
@@ -18,7 +18,6 @@ def __init__(self, **kwargs):
         self._attr_names += [
             "identifier",
             "src",
-            "meta",
```

Comment:
Now we are getting image resolution directly from the base64 image in `ImageDirection.vue`.  I don't think we are using `state.[meta_id]` anywhere if we want to clean that code out.

Reply:
I'll do something with this `state.[meta_id]` code in my work on "metrics" per image